src/Lecture3.hs

```diff
@@ -229,8 +265,16 @@ types that can have such an instance.
 -- instance Functor Weekday where
 -- instance Functor Gold where
 -- instance Functor Reward where
--- instance Functor List1 where
--- instance Functor Treasure where
+instance Functor List1 where
+    -- Just for me
+    -- fmap :: (a -> b) -> f a -> f b
```

Comment:
You don't need to comment type signature, you can use `InstanceSigs` and write the correct type signature directly here 🙂

Reply:
Yeah, also need to practise with it more.